### ISSUE TITLE
Merge monitor switch and monitor brightness in a light entity

### ISSUE THREAD

**Issue by sisimomo** [sindrebroch/ha-magicmirror#12]
**Describe the solution you'd like**
It would be simpler if the monitor switch and the brightness were merge in a single entity (light).
![image](https://user-images.githubusercontent.com/27745204/174386066-663d3d5e-af9b-4a6e-9cb3-11c940e1db83.png)

**Describe alternatives you've considered**
Here's the Template Light I'm currently using to get the wanted result:
```yaml
light:
  - platform: template
    lights:
      magicmirror_monitor:
        friendly_name: "MagicMirror Monitor"
        icon_template: mdi:mirror-rectangle
        level_template: >-
          {% if is_state('switch.magicmirror_monitor', 'on')|default(true, false) %}
            {{ ((states('number.magicmirror_brightness')|default(100, false)|int)*2.55)|round() }}
          {% else %}
            0
          {% endif %}
        value_template: "{{ states('switch.magicmirror_monitor')|default('on', false) }}"
        turn_on:
          service: switch.turn_on
          target:
            entity_id: switch.magicmirror_monitor
        turn_off:
          service: switch.turn_off
          target:
            entity_id: switch.magicmirror_monitor
        set_level:
          service: number.set_value
          data:
            value: "{{ (brightness/2.55)|round() }}"
          target:
            entity_id: number.magicmirror_brightness
```

**Comment by sindrebroch** [comment]
Great suggestion! Will definetely implement this at some point.

**Comment by sindrebroch** [comment]
Should be fixed in release 1.4.0. Please reopen if there are any issues.
